### PR TITLE
Fix formatting

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

**Comment by pranavkm** [comment]
@karolz-ms - could you have a look at the second commit - https://github.com/aspnet/Logging/pull/448/commits/00053f11f424ae0f5415e04742fb869324754e60?

**Comment by karolz-ms** [comment]
@pranavkm Looks good, thanks a bunch!!

File: src/Microsoft.Extensions.Logging.EventSource/EventSourceLoggerProvider.cs [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+            var specPos = 0;
+            for (;;)
+            {
+                if (namePos < loggerName.Length)
+                {
+                    if (filterSpec.Length <= specPos)
+                    {
+                        break;
+                    }
+                    char specChar = filterSpec[specPos++];
```
Comment (Eilon) [inline src/Microsoft.Extensions.Logging.EventSource/EventSourceLoggerProvider.cs]: var

File: src/Microsoft.Extensions.Logging.EventSource/EventSourceLoggerProvider.cs [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+                    char nameChar = loggerName[namePos++];
+                    if (specChar == nameChar)
+                    {
+                        continue;
+                    }
+
+                    // We allow wildcards a the end.
```
Comment (Eilon) [inline src/Microsoft.Extensions.Logging.EventSource/EventSourceLoggerProvider.cs]: *at* the end?

**Comment by Eilon** [comment]
:shipit:

File: src/Microsoft.Extensions.Logging.EventSource/project.json [hunk 3]
```diff
@@ -0,0 +1,42 @@
+{
+  "version": "1.0.0-*",
```
Comment (Eilon) [inline src/Microsoft.Extensions.Logging.EventSource/project.json]: I think change this to `1.1.0` at this point.

File: src/Microsoft.Extensions.Logging.EventSource/project.json [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    }
+  },
+  "buildOptions": {
+    "warningsAsErrors": true,
+    "keyFile": "../../tools/Key.snk",
+    "xmlDoc": true
+  },
+  "dependencies": {
+    "Microsoft.Extensions.Logging.Abstractions": "1.0.0-*",
+    "Newtonsoft.Json": "9.0.1-beta1"
```
Comment (Eilon) [inline src/Microsoft.Extensions.Logging.EventSource/project.json]: 9.0.1 RTM

**Comment by karolz-ms** [comment]
@pranavkm @Eilon Cool, thx guys, let me know if I can help with anything

**Comment by pranavkm** [comment]
Like we discussed offline, we have to tweak how we build `net4x` in xplat. I'm going to work on that this week.

**Pull request merged**